editor/editor_node.cpp

```diff
@@ -58,6 +58,7 @@
 #include "servers/physics_2d_server.h"
 #include "translation.h"
 #include "version.h"
+#include "version_hash.gen.h"
```

Comment:
Might be nice to split the about dialog in its own file to avoid rebuilding the whole of editor_node each time.